backend/tests/test_api.py

```diff
@@ -18,37 +19,101 @@ def test_hello(client):
     assert response.json() == {"message": "Hello user! Tip: open /docs or /redoc for documentation"}
 
 
-def test_delete_project(neo4j, client):
-    session = neo4j.session()
-    create_project = """
-        CREATE (n:PROJECT)
-        SET n.id = 'test_project'
-        SET n.taxonomy_name = 'test_taxonomy_name'
-        SET n.branch = 'test_branch'
-        SET n.description = 'test_description'
-        SET n.status = 'OPEN'
-        SET n.project_name = 'p_test_taxonomy_name_test_branch_name'
-        SET n.created_at = datetime()
-    """
-
-    create_project2 = """
-        CREATE (n:PROJECT)
-        SET n.id = 'test_project2'
-        SET n.taxonomy_name = 'test_taxonomy_name2'
-        SET n.branch = 'test_branch2'
-        SET n.description = 'test_description2'
-        SET n.status = 'OPEN'
-        SET n.project_name = 'p_test_taxonomy_name_test_branch_name2'
-        SET n.created_at = datetime()
-    """
-    session.run(create_project)
-    session.run(create_project2)
-
-    response = client.delete("/test_taxonomy_name/test_branch/delete")
+def test_ping(client):
+    response = client.get("/ping")
     assert response.status_code == 200
-    assert response.json() == {"message": "Deleted 1 projects"}
+    assert response.json().get("ping").startswith("pong @")
+
+
+def test_import_from_github(client, mocker):
+    mocker.patch("editor.api.TaxonomyGraph.is_branch_unique", return_value=True)
+    mocker.patch("editor.api.TaxonomyGraph.import_from_github", return_value=True)
```

Comment:
@alexgarel please tell me what you think about how I mocked the calls to Github (I struggle to mock directly the GithubOperations class). 
Also, should I create the corresponding integration tests which use the real Github servers?

Reply:
As seen together:

* we will patch github.Github instead
* replace all `from github import Github` by `import github` and `Github` by `github.Github` so that mocking once affect all use of it.